project/docker/services/web/Dockerfile.production

```diff
@@ -24,6 +24,7 @@ RUN set -ex                                                 && \
       git                                                      \
       libmagickwand-dev                                        \
       libpq-dev                                                \
+      subversion                                               \ 
```

Comment:
Is this really necessary? Any plugin or dependency that needs to be cloned via svn?

Reply:
Yes, it is useful when you want to test the repository module from Redmine.